src/ws/Reply.ts

```diff
@@ -1,22 +1,30 @@
+import { ErrorCode } from '../errors';
+
 export interface IPacket {
-    error: string;
-    data: any;
+  error: string | IErrorPacket;
+  data: any;
+}
+
+export interface IErrorPacket {
+  code: ErrorCode;
+  message: string;
+  stacktrace: string;
```

Comment:
We should probably never be returning stacktraces from inside the chat server?

Reply:
Yeeaaa that shouldn't be the case *al all*.

Reply:
That what we get for some reason
cc: @dadeside ^